roles/db_integrations/tasks/main.yml

```diff
@@ -26,6 +26,7 @@
     - "GRANT SELECT ON ALL SEQUENCES IN SCHEMA public TO {{ db_integration.user }};"
     - "ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO {{ db_integration.user }};"
     - "ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON SEQUENCES TO {{ db_integration.user }};"
+    - "ALTER ROLE {{ db_integration.user }} SET statement_timeout = $$30s$$;"
```

Comment:
> This new option is only executed when a user is added and therefore
existing users are not affected. New users like n8n and metabase will be
restricted.

I'm wondering what happens if we provision a new instance? All user will have this 30s timeout? You don't think it can be a problem?

Reply:
This only applies to the db_integrations. That's zapier, metabase and n8n. I think that all of them should be restricted to avoid downtime of the app.